examples/non-kubernetes/aws-otel-config-file.yaml

```diff
@@ -32,11 +20,13 @@ processors:
     override: true
 exporters:
   otlphttp:
-    endpoint: $SUMO_HTTP_TRACES_URL
+    traces_endpoint: SUMO_HTTP_TRACES_URL
```

Comment:
I am thinking now we should use `endpoint`. The reason being: it will make everything simpler after we do Tracing source->OTLP source migration